tests/020_normal_progress.test

```diff
@@ -35,5 +35,11 @@ $FWUP_APPLY -a -d $IMGFILE -i $FWFILE -t complete | $SED 's/\r/./g' > $WORK/actu
 cat >$WORK/expected_output.txt <<EOF
 .  0%. 25%. 50%. 75%. 99%.100%
 EOF
+
+# There's one more CR in the output under Wine due to the line-ending
```

Comment:
Yeah, I agree. In this case, it didn't work because the `\r` characters are getting mapped to `.` characters, but Windows formats line endings as `\r\n` instead of just `\n`, so there's an extra dot instead of extra white space. I wasn't super happy with how complicated it was to ignore that trailing dot, so feel free to suggest something else if you think of something.